src/__experimental__/components/simple-color-picker/simple-color-picker.stories.js

```diff
@@ -74,6 +74,59 @@ function makeStory(storyName, themeSelector) {
   return [storyName, component, metadata];
 }
 
+function makeValidationsStory(storyName, themeSelector) {
```

Comment:
@harpalsingh Please review
![image](https://user-images.githubusercontent.com/2328042/80135071-94217d00-8597-11ea-8aa6-dc334f52317a.png)

Reply:
Also, it doesn't work when it's a boolean, but the prop types say it should.